publisher/publisher_test.go

```diff
@@ -8,6 +8,51 @@ import (
 	"github.com/honeycombio/honeytail/event"
 )
 
+func TestParseTraceData(t *testing.T) {
+	testCases := []struct {
+		ev       event.Event
+		expected map[string]interface{}
+	}{
+		{
+			ev: event.Event{
+				Data: map[string]interface{}{
+					"trace_id":                "Root=1-5759e988-bd862e3fe1be46a994272793;Parent=53995c3f42cd8ad8;Self=1-67891234-12456789abcdef012345678;Sampled=1",
+					"request_processing_time": 1.12,
+					"elb":          "fooservice",
+					"request_path": "/down/stream",
+				},
+			},
+			expected: map[string]interface{}{
+				// old fields
+				"trace_id":                "Root=1-5759e988-bd862e3fe1be46a994272793;Parent=53995c3f42cd8ad8;Self=1-67891234-12456789abcdef012345678;Sampled=1",
```

Comment:
confused about these field names. We use `trace_id` as a special field in our trace UI. Won't this get picked up as the "root" span?

Reply:
It's both `trace_id` and `traceId` I think :| I wonder if I should modify it when the trace fields are parsed